src/assets/images/reacthn.svg

```diff
@@ -0,0 +1,10 @@
tYJUCAAAECBAgQIECAAAECBBILCEwSD1/rBAgQIECAAAECBAgQIECAQLOAwKTZxSoBAgQIECBAgAABAgQIECCQWEBgknj4WidAgAABAgQIECBAgAABAgSaBQQmzS5WCRAgQIAAAQIECBAgQIAAgcQCApPEw9c6AQIECBAgQIAAAQIECBAg0CwgMGl2sUqAAAECBAgQIECAAAECBAgkFhCYJB6+1gkQIECAAAECBAgQIECAAIFmAYFJs4tVAgQIECBAgAABAgQIECBAILGAwCTx8LVOgAABAgQIECBAgAABAgQINAsITJpdrBIgQIAAAQIECBAgQIAAAQKJBQQmiYevdQIECBAgQIAAAQIECBAgQKBZQGDS7GKVAAECBAgQIECAAAECBAgQSCwgMEk8fK0TIECAAAECBAgQIECAAAECzQICk2YXqwQIECBAgAABAgQIECBAgEBiAYFJ4uFrnQABAgQIECBAgAABAgQIEGgWEJg0u1glQIAAAQIECBAgQIAAAQIEEgsITBIPX+sECBAgQIAAAQIECBAgQIBAs4DApNnFKgECBAgQIECAAAECBAgQIJBYQGCSePhaJ0CAAAECBAgQIECAAAECBJoFBCbNLlYJECBAgAABAgQIECBAgACBxAICk8TD1zoBAgQIECBAgAABAgQIECDQLCAwaXaxSoAAAQIECBAgQIAAAQIECCQWEJgkHr7WCRAgQIAAAQIECBAgQIAAgWYBgUmzi1UCBAgQIECAAAECBAgQIEAgsYDAJPHwtU6AAAECBAgQIECAAAECBAg0CwhMml2sEiBAgAABAgQIECBAgAABAokFBCaJh691AgQIECBAgAABAgQIECBAoFlAYNLsYpUAAQIECBAgQIAAAQIECBBILCAwSTx8rRMgQIAAAQIECBAgQIAAAQLNAgKTZherBAgQIECAAAECBAgQIECAQGIBgUni4WudAAECBAgQIECAAAECBAgQaBYQmDS7WCVAgAABAgQIECBAgAABAgQSCwhMEg9f6wQIECBAgAABAgQIECBAgECzgMCk2cUqAQIECBAgQIAAAQIECBAgkFhAYJJ4+FonQIAAAQIECBAgQIAAAQIEmgUEJs0uVgkQIECAAAECBAgQIECAAIHEAgKTxMPXOgECBAgQIECAAAECBAgQINAsIDBpdrFKgAABAgQIECBAgAABAgQIJBYQmCQevtYJECBAgAABAgQIECBAgACBZgGBSbOLVQIECBAgQIAAAQIECBAgQCCxgMAk8fC1ToAAAQIECBAgQIAAAQIECDQLCEyaXawSIECAAAECBAgQIECAAAECiQUEJomHr3UCBAgQIECAAAECBAgQIECgWUBg0uxilQABAgQIECBAgAABAgQIEEgsIDBJPHytEyBAgAABAgQIECBAgAABAs0CApNmF6sECBAgQIAAAQIECBAgQIBAYgGBSeLha50AAQIECBAgQIAAAQIECBBoFhCYNLtYJUCAAAECBAgQIECAAAECBBILCEwSD1/rBAgQIECAAAECBAgQIECAQLOAwKTZxSoBAgQIECBAgAABAgQIECCQWEBgknj4WidAgAABAgQIECBAgAABAgSaBQQmzS5WCRAgQIAAAQIECBAgQIAAgcQCApPEw9c6AQIECBAgQIAAAQIECBAg0CwgMGl2sUqAAAECBAgQIECAAAECBAgkFhCYJB6+1gkQIECAAAECBAgQIECAAIFmAYFJs4tVAgQIECBAgAABAgQIECBAILGAwCTx8LVOgAABAgQIECBAgAABAgQINAsITJpdrBIgQIAAAQIECBAgQIAAAQKJBQQmiYevdQIECBAgQIAAAQIECBAgQKBZQGDS7GKVAAECBAgQIECAAAECBAgQSCwgMEk8fK0TIECAAAECBAgQIECAAAECzQICk2YXqwQIECBAgAABAgQIECBAgEBiAYFJ4uFrnQABAgQIECBAgAABAgQIEGgWEJg0u1glQIAAAQIECBAgQIAAAQIEEgsITBIPX+sECBAgQIAAAQIECBAgQIBAs4DApNnFKgECBAgQIECAAAECBAgQIJBYQGCSePhaJ0CAAAECBAgQIECAAAECBJoFBCbNLlYJECBAgAABAgQIECBAgACBxAICk8TD1zoBAgQIECBAgAABAgQIECDQLCAwaXaxSoAAAQIECBAgQIAAAQIECCQWWNV572fUnR/hAAIECBAgQIAAAQIECBAgQIBAmwLeYdKmpr0IECBAgAABAgQIECBAgACBUQgITEYxRk0QIECAAAECBAgQIECAAAECbQoITNrUtBcBAgQIECBAgAABAgQIECAwCgGBySjGqAkCBAgQIECAAAECBAgQIECgTQGBSZua9iJAgAABAgQIECBAgAABAgRGISAwGcUYNUGAAAECBAgQIECAAAECBAi0KSAwaVPTXgQIECBAgAABAgQIECBAgMAoBAQmoxijJggQIECAAAECBAgQIECAAIE2BQQmbWraiwABAgQIECBAgAABAgQIEBiFgMBkFGPUBAECBAgQIECAAAECBAgQINCmgMCkTU17ESBAgAABAgQIECBAgAABAqMQEJiMYoyaIECAAAECBAgQIECAAAECBNoUEJi0qWkvAgQIECBAgAABAgQIECBAYBQCApNRjFETBAgQIECAAAECBAgQIECAQJsCApM2Ne1FgAABAgQIECBAgAABAgQIjEJAYDKKMWqCAAECBAgQIECAAAECBAgQaFNAYNKmpr0IECBAgAABAgQIECBAgACBUQgITEYxRk0QIECAAAECBAgQIECAAAECbQoITNrUtBcBAgQIECBAgAABAgQIECAwCgGBySjGqAkCBAgQIECAAAECBAgQIECgTQGBSZua9iJAgAABAgQIECBAgAABAgRGISAwGcUYNUGAAAECBAgQIECAAAECBAi0KSAwaVPTXgQIECBAgAABAgQIECBAgMAoBP4/sdh8XCm4MrwAAAAASUVORK5CYII="></image>
```

Comment:
[Found one](https://github.com/insin/react-hn/blob/master/public/img/safari-pinned-tab.svg) with actual curves, could you please update?